storages/backends/dropbox.py

```diff
@@ -158,3 +166,24 @@ def _chunked_upload(self, content, dest_path):
                     content.read(self.CHUNK_SIZE), cursor
                 )
                 cursor.offset = content.tell()
+
+    def _clean_name(self, name):
```

Comment:
I think you can just use `self._full_path` instead here since that seems to work.

Reply:
done